lib/torrent.js

```diff
@@ -582,6 +584,10 @@ class Torrent extends EventEmitter {
   }
 
   _verifyPieces (cb) {
+    const done = () => {
+      this._checkDone()
```

Comment:
Why do we need to call it here?

I think we are already calling it in the callback when `_verifyPieces` is called.

Reply:
It does not do it through the codepath of onMetadata where skip verify is false

Reply:
would you rather me add it there or remove it from rescan cb?